iOS-13-Migration-Guide.md

```diff
@@ -1,8 +1,48 @@
 # Migrating to iOS 13
 
-## iOS 13 & Xcode 11 Support
```

Comment:
Please add a note about how our latest findings indicate that we found it's not necessarily Xcode 11